backend/tournesol/tests/test_api_comparison.py

```diff
@@ -1371,6 +1372,7 @@ def setUp(self):
 
         self.client = APIClient()
 
+    @pytest.mark.skip("Online updates not implemented in Solidago")
```

Comment:
We could use the annotation `skip` from the module `unittest` instead, if we want both `python manage.py test` and `pytest` to skip the test.